tuxemon/core/event/conditions/variable_set.py

```diff
@@ -54,21 +53,22 @@ def test(self, session,  condition):
             "parameters": [
                 "battle_won:yes"
             ],
-            "width": 1,
-            "height": 1,
-            "operator": "is",
-            "x": 2,
-            "y": 2,
             ...
         }
 
         """
         player = session.player
 
-        # Split the string by ":" into a list
-        key, value = condition.parameters[0].split(":")
+        parts = condition.parameters[0].split(":")
```

Comment:
looks like now it will return True if you have a condition but don't specify the value "variable_set variable".  If thats the case please update the docstring.

Reply:
Already did. In fact, the existing docstring was incorrect, but all the current usages depend on the fact it checks the value, so I could only extend the functionality instead of "correcting" it to match the docstring.